LOGIC_GATES/LOOPS/APPEND/APPEND.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from flojoy import flojoy, DataContainer
+
+
+@flojoy
+def APPEND(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """The APPEND node appends a single data point to an array.
+    The large array must be passed to the bottom "array" connection.
+    The single point must have a shape of 1 (or (1,)).
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    Ordered pair (numpy arrays).
+    """
+
+    if len(dc_inputs) < 2:
+        raise ValueError(
+            f"To append, APPEND node requires two inputs, {len(dc_inputs)} was given!"
+        )
+    y0 = dc_inputs[0].y
```

Comment:
This assumes that the data containers will have `x` and `y` members, which I think means they're `ordered_pairs`? Could you make this node more generic, and have it potentially work with type matrix and or dataframe?

Reply:
I think this will probably be a different node (CONCAT) to work with pd.concat and np.concatenate.